Libraries/LibWeb/DOM/Document.h

```diff
@@ -23,6 +23,7 @@
 #include <LibUnicode/Forward.h>
 #include <LibWeb/CSS/CSSStyleSheet.h>
 #include <LibWeb/CSS/StyleSheetList.h>
+#include <LibWeb/CSS/CSSPropertyRule.h>
 #include <LibWeb/Cookie/Cookie.h>
```

Comment:
Make sure to run `clang-format` on your changes. That'll put these in alphabetical order, among other things. You can make that run automatically by installing the commit hook: https://github.com/LadybirdBrowser/ladybird/blob/master/CONTRIBUTING.md#commit-hooks